CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @neondatabase/developer-productivity
```

Comment:
Probably, it should be something like this:

```suggestion
/ @neondatabase/developer-productivity
```

Reply:
![image](https://github.com/user-attachments/assets/a0f90a02-2c91-4218-abd0-e6b4eb6035bd)

Reply:
that didn't seem to be the problem...

Reply:
Ah, we probably should use `./**`, found it here: https://stackoverflow.com/a/78110524

Reply:
> > make sure the team @neondatabase/developer-productivity exists, is publicly visible, and has write access to the repository
> 
> @bayandin can you ensure these are all true?

Hmm, we use this team in other repos and it works fine (for example, in [neon](https://github.com/neondatabase/neon/blob/6f6749c4a96673430fa6ecce26ca756ead7e8a46/CODEOWNERS#L1)), that's why I assumed that the problem with `*`, but it mentioned [in docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file).